ocw-course-v2/ocw-studio.yaml

```
@@ -274,6 +275,67 @@ collections:
             name: archive_url
             widget: string
 
+  - category: Content
+    name: external-resource
+    label: External Resources
+    folder: content/external-resources
+    fields:
+      - label: Title
+        name: title
+        widget: string
+        required: true
+        help: >
+          Text to display for the link.
+          You can change this later when adding links.
+
+      - label: URL
+        name: external_url
+        widget: string
+        required: true
+        help: >
+          The URL. For example, https://raise.mit.edu/
+          or https://owl.english.purdue.edu/owl/resource/747/01/
```

```suggestion
        help: >
          The URL. For example, https://owl.english.purdue.edu/owl/resource/747/01/
```

on example is enough